tests/data/codelist/tier_attribute/valid/variables.yaml

```diff
@@ -0,0 +1,11 @@
+- Primary Energy:
+    description: Total primary energy consumption
+    unit: EJ/yr
+- Primary Energy|{Fuel}|{Sector}:
+    description: Primary energy consumption of {Fuel}
+    unit: EJ/yr
+    tier: 1
+- Share|{Fuel}:
+    description: Share of {Fuel} in the total primary energy mix
+    unit: '%'
+    tier: 2
```

Comment:
This variable should be called "Primary Energy|{Fuel} [Share]" according to the conventions, provided another suggestion above.
```suggestion
```